apps/web/src/app/(dashboard)/documents/upload-document.tsx

```diff
@@ -96,10 +96,12 @@ export const UploadDocument = ({ className }: UploadDocumentProps) => {
     }
   };
 
+  const isSmallVerticalScreen = typeof window !== 'undefined' && window.innerHeight < 800;
+
   return (
     <div className={cn('relative', className)}>
       <DocumentDropzone
-        className="min-h-[40vh]"
+        className={`${isSmallVerticalScreen ? 'min-h-[50vh]' : 'min-h-[40vh]'}`}
```

Comment:
I reckon we could something closer to `min-h-[max(400px,40vh)` or similar to achieve the desired effect without javascript 😄

Reply:
Actually i want to set the min height to 50% of the vh in smaller vertical screens, and not a constant number like 400px here. Plus i tried doing ```min-h-[max(400px,40vh)]```, it didn't worked!

Other thing we could do is set the min height to ```50vh``` instead of ```40vh``` for all screen sizes.

Reply:
You were absolutely right, it should have been a `min` rather than `max`. I've updated and tested across a variety of screen sizes!

This has the added benefit of not making the upload massive on vertical monitors which is nice 😄